src/sync.js

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+import Cache from "./Cache.js";
+
+const sync = (async () => {
+    const response = await axios.get(
+        "http://worldtimeapi.org/api/timezone/gmt"
```

Comment:
After looking into WorldTimeAPI, it looks like it is not recommended for production environments. ([source](http://worldtimeapi.org/pages/faqs#please-do-not))

It turns out that NTP servers [use the UDP communications protocol](https://datatracker.ietf.org/doc/html/rfc5905#page-18), which excludes being able to communicate with them via the browser, ruling direct NTP communication out as a solution.

Following our discussion, grabbing the `Date` header off a response from [time.google.com](time.google.com) looks to be a good workaround.